unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -807,6 +825,7 @@ class SSTKernelHex8Mesh : public LowMachKernelHex8Mesh
       *bulk_, *coordinates_, *density_);
     unit_test_kernel_utils::tke_test_function(*bulk_, *coordinates_, *tke_);
     unit_test_kernel_utils::sdr_test_function(*bulk_, *coordinates_, *sdr_);
+    unit_test_kernel_utils::gamint_test_function(*bulk_, *coordinates_, *gamint_); // added
```

Comment:
here as well. Comments like `// added` should be removed everywhere.

Reply:
done